pr_agent/algo/pr_processing.py

```diff
@@ -29,10 +29,10 @@ def get_pr_diff(git_provider: Union[GithubProvider, Any], token_handler: TokenHa
         global PATCH_EXTRA_LINES
         PATCH_EXTRA_LINES = 0
 
-    git_provider.pr.diff_files = list(git_provider.get_diff_files())
+    diff_files = list(git_provider.get_diff_files())
```

Comment:
**Suggestion:** Consider using a more descriptive variable name than `diff_files`. It's not clear what the difference is between the files. A more descriptive name could be `changed_files` or `modified_files`.
```suggestion
    changed_files = list(git_provider.get_diff_files())
```